### PR TITLE
Add .travis.yml for building

### PR DESCRIPTION
**Pull request by raboof**
Not yet for deploying

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: scala
+
+scala:
+  - "2.11.8"
+  - "2.12.1"
```
Comment (2m) [inline .travis.yml]: 2.11.11 and 2.12.2 :)

Comment (raboof) [inline .travis.yml]: or actually 2.10.6 as we're building an sbt plugin, I think, right?

File: version.sbt [hunk 2]
```diff
@@ -1 +0,0 @@
-version in ThisBuild := "0.1.0-SNAPSHOT"
```
Comment (pvlugter) [inline version.sbt]: The `version.sbt` is used by sbt-release. Is this not needed?

Comment (pvlugter) [inline version.sbt]: Or it's just released via travis deploy now?

Comment (2m) [inline version.sbt]: We are aiming to having it released by creating a tag in Github and then Travis doing all the work. Therefore it is not really possible to have changes being made in `version.sbt` during the release, as then Travis would need to have commit rights.

Version will be derived from the `git describe` by the `sbt-dynver` and therefore this file is removed.

Comment (pvlugter) [inline version.sbt]: Cool. That sounds good.

**Pull request merged**